test/integration/consul-container/libs/service/connect.go

```diff
@@ -55,6 +55,10 @@ func (g ConnectContainer) GetAddr() (string, int) {
 	return g.ip, g.appPort
 }
 
+func (g ConnectContainer) GetPort(port int) int {
+	return 0
+}
+
```

Comment:
Is this a placeholder value?

Reply:
I can address that since I added it 😄 -- it's to make the `ConnectContainer` adhere to the `Service` interface in this package since I added a `GetPort` method in order to retrieve the ports that we dynamically added to the punched port mappings in the proxy container that acts as the entrypoint to all the docker containers that get run (you need to map from the port you asked for it to forward to your container to the port that it's actually mapped to on the host in order to route to it).